### DISCUSSION TITLE
Adding other or bespoke characterisation factors

### DISCUSSION THREAD

**Discussion by alyabolowich** [LCA-ActivityBrowser/activity-browser#1276] [Q&A] [answered]
Hi,

I have checked the Discussion and Issues to see how it is possible to add custom characterization factors to biosphere flows? I saw this discussion here from 2022 ([https://brightway.groups.io/g/AB-discussion/topic/94117759](https://brightway.groups.io/g/AB-discussion/topic/94117759)), but unfortunately I have not been able to figure out how to do this in the AB (I'm using v2.9.7). 

Any ideas how to add custom characterization factors to the biosphere in AB?

Thank you!
Alya

**Comment by alyabolowich** [comment]
I see it's possible to go into the Impact Categories tab, click on the method you want, and then double click the impact category you want. This will bring up the characterization factors (CFs) and they can be edited here by clicking "Edit Characterization Factors". 

This helps me solve one problem (modifying a characterization factor for an impact category), but I'd also like to know how to add new characterization factors from different methods (say for regionalization). 

![image](https://github.com/LCA-ActivityBrowser/activity-browser/assets/54064746/8d6e4bad-2171-4698-835b-28149c643d9b)

Thanks again!

**Reply by marc-vdm** [reply, marked as answer]
Hi Alya,

Working with impact categories (ICs) in AB is currently (v `2.9.7`) still limited. You can do the following:
- Duplicate (groups of) ICs from the tree/list on the left
- Edit characterization factors of ICs (like you show in the screenshot)
- Add characterization factors by dragging them from the Biosphere into the IC, then edit the number to what you want
- Remove characterization factors from an IC by right-click > Delete
- Remove (groups of) ICs right-click on the tree/list on the left and > Delete (warning: this cannot be undone!)

In the future we will expand these features and allow for a better way of creating ICs. We'd also be happy to discuss importing full ICs, but there's no standardized format AFAIK, which could make this very complex, which could keep this more a Brightway feature instead of an AB feature.

Hope this answers your question!

**Reply by alyabolowich** [reply]
Thanks Marc!! :+1:
